readme.md

```diff
@@ -1,11 +1,88 @@
-# [WIP] Spade Tenant Service
+<div align=center>
+<img src="docs/logo.png" width=300 />
 
+# Spade Tenant Services
+</div>
 
-## Generating Schema
-`DB_ALLOW_MIRGATIONS=true go run . > out.sql`
 
+Tenant APIs and services for the [Spade](https://github.com/data-preservation-programs/spade) - Filecoin network storage proposal management service
 
-## Generating Swagger
+The Tenant services handle all aspects of Spade Storage Proposals from the Tenant side. At a high level, this includes interfaces to manage the following:
+- Policies
+- Storage Contracts
+- Collections
+- Content (i.e, sources of Filecoin Pieces)
+- Replication Constraints
+- Storage Provider SLA Compliance
+
+A high level architecture diagram of the Tenant functionalities and their relationships can be found [here](https://miro.com/app/board/uXjVMwfo8TM=/?share_link_id=267050417898) 
+
+<img src="docs/arch-diagram.png" width=500 />
+
+# API
+All functionality is exposed over a REST API. Live API docs can be viewed at https://spade.storage (Coming soon), or inside the repository [docs folder.](https://github.com/data-preservation-programs/spade-tenant/tree/main/docs)
```

Comment:
Should we include this with everything that is going on

Reply:
good point - will reword to make it more open-ended